cves/CVE-2005-2728.yml

```diff
@@ -31,7 +31,10 @@ description_instructions: |
   that outsiders to Chromium would not understand. Technology like "regular
   expressions" is fine, and security phrases like "invalid write" are fine to
   keep too.
-description:
+description: |
+  There is a non-limited byte-range weakness that can cause a denial of service attack. If an attacker
```

Comment:
Could you explain more about what a byte-range weakness is?

Reply:
Good idea. I did just kind of brush past that originally. Added more description. Thank you!